chains/solana/idl.go

```diff
@@ -0,0 +1,11 @@
+package solana
+
+import _ "embed"
+
+//go:embed contracts/target/idl/ccip_router.json
+var ccipRouter string
```

Comment:
can we name this as `ccipRouterIdl`? the name is a bit generic and could cause issues in the future